test-mcp.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "Testing MCP server..."
+
+# Start the MCP server in the background
+cd /home/runner/work/trails/trails
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)